wire-library/wire-grpc-client/src/commonMain/kotlin/com/squareup/wire/GrpcResponse.kt

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2020 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@file:JvmName("GrpcResponseCloseable")
```

Comment:
Is there a problem if we don't rename it?

Reply:
Yes, the compiler generates two class files (for `GrpcResponse` and for top-level functions) with the same name and fails the build.